training/src/main/java/com/wolox/training/dto/BookResponseDto.java

```diff
@@ -0,0 +1,38 @@
+package com.wolox.training.dto;
+
+import com.wolox.training.models.Book;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class BookResponseDto {
```

Comment:
This class is not necessary, Camel can transform the Book instance into JSON and vice versa